### DISCUSSION TITLE
converting datetime.datetime to str

### DISCUSSION THREAD

**Discussion by micjgam** [kevinzg/facebook-scraper#642] [Q&A] [answered]
So I am able to pull the datetime.datetime value out of specific posts, but am having trouble converting to a string which I need to be able to text to myself.

    for post in get_posts(group='group ID', credentials=(email, password), pages=1):
        keyword = WHATEVER IT NEEDS TO BE
        if keyword in post['post_text']:
            date = post['timestamp']
            date = datetime.fromtimestamp(date)
            date.strftime("%m/%d %H:%M")

After this bit of code, I am still getting that type(date) is <class 'datetime.datetime'>.

What am I doing wrong here?

**Comment by neon-ninja** [comment, marked as answer]
Why not do `str(post['time'])`?

**Reply by micjgam** [reply]
Wow... was I just overcomplicating things?  I am relatively new to coding and sometimes the simple answers elude me.

THANKS!
